template/go-ofwatchdog/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM golang:1.10.4-alpine3.8 as builder
+
+# Allows you to add additional packages via build-arg
+ARG ADDITIONAL_PACKAGE
```

Comment:
Could you please remove this? I believe you used to experiment, but this idea breaks one important property: hermeticity. One would be able to change the build environment throught vars.

Reply:
Sorry, I merged it too quickly. In fact, that addition of mine. All go official openfaas templates (except armhf version) uses it.